tests/foreman/performance/common/perf_candlepin.py

```diff
@@ -0,0 +1,94 @@
+from robottelo.common import conf, ssh
+
+import logging
+import time
+import requests
+
+class CandlepinPerfTest:
```

Comment:
`robottelo.test.TestCase` sets a default logger and enables "long" error messages (a *very* handy setting that is the default in Python 3). Consider inheriting from that class, and take a look through that module to see how some of the other test case classes work. They may provide some inspiration.

At the very least, inherit from `object`:

    class CandlepinPerfTest(object):

Not inheriting from `object` is - what - an old-style class?